packages/Modal/Modal.jsx

```diff
@@ -56,17 +56,6 @@ const Modal = ({
     return null
   }
 
-  const preventScroll = e => {
-    if (ModalOverlayRef.current.contains(e.targetTouches[0].target)) {
-      e.preventDefault()
-    }
-  }
-
-  const removeEventScrolling = () => {
-    document.body.removeEventListener('touchmove', preventScroll)
-    document.body.style.overflow = 'visible'
-  }
-
```

Comment:
Hey,
Thanks for submitting the PR. We validated the changes to the scroll behaviour on mobile, but after the modal is closed, the page does not scroll.  Suspecting it's the overflow property on the body that's affecting this behaviour.

I believe the `removeEventScrolling` should be kept and called in the hook and the removeEventListener on the 'touchmove' event be removed.

Should be good for approval after that. Thanks.